test/Microsoft.TestPlatform.AcceptanceTests/Extension/NetCoreTargetFrameworkDataSourceAttribute.cs

```diff
@@ -35,15 +34,12 @@ public class NetCoreTargetFrameworkDataSourceAttribute : Attribute, ITestDataSou
         bool useDesktopRunner = true,
         // adding another runner is not necessary until we need to start building against another
         // sdk, because the netcoreapp2.1 executable is forward compatible
+        // REVIEW ME: Does this comment needs to be updated?
```

Comment:
```suggestion
```

Not needed now that the runner and the only host are targeting the same runtime version.